sql/core/src/test/resources/sql-tests/results/csv-functions.sql.out

```diff
@@ -15,7 +15,7 @@ select from_csv('26/08/2015', 'time Timestamp', map('timestampFormat', 'dd/MM/yy
 -- !query 1 schema
 struct<from_csv(26/08/2015):struct<time:timestamp>>
 -- !query 1 output
-{"time":2015-08-26 00:00:00.0}
+{"time":2015-08-26 00:00:00}
```

Comment:
which causes this change?

Reply:
it is toString before

Reply:
Can you point out the line number?

Reply:
you mean https://github.com/apache/spark/pull/26942/files#diff-303a0a0e4383242d22307edcdd82e1f1L101?

Reply:
Dose this match with Hive string? I think this was intentionally matched to Hive results.

Reply:
After this patch, yes. Please see hive's result below:
```
0: jdbc:hive2://localhost:10000> with t as (select '28/04/2017 00:00:00' as mydate)
0: jdbc:hive2://localhost:10000> select  array(from_unixtime(to_unix_timestamp (mydate,'dd/MM/yyyy HH:mm:ss')))
0: jdbc:hive2://localhost:10000> from    t;
INFO  : Compiling command(queryId=hive_20191220051919_7c63ab36-cc5a-42c7-990a-fd66fb8d609d): with t as (select '28/04/2017 00:00:00' as mydate)
select  array(from_unixtime(to_unix_timestamp (mydate,'dd/MM/yyyy HH:mm:ss')))
from    t
INFO  : Semantic Analysis Completed
INFO  : Returning Hive schema: Schema(fieldSchemas:[FieldSchema(name:_c0, type:array<string>, comment:null)], properties:null)
INFO  : Completed compiling command(queryId=hive_20191220051919_7c63ab36-cc5a-42c7-990a-fd66fb8d609d); Time taken: 0.077 seconds
INFO  : Concurrency mode is disabled, not creating a lock manager
INFO  : Executing command(queryId=hive_20191220051919_7c63ab36-cc5a-42c7-990a-fd66fb8d609d): with t as (select '28/04/2017 00:00:00' as mydate)
select  array(from_unixtime(to_unix_timestamp (mydate,'dd/MM/yyyy HH:mm:ss')))
from    t
INFO  : Completed executing command(queryId=hive_20191220051919_7c63ab36-cc5a-42c7-990a-fd66fb8d609d); Time taken: 0.0 seconds
INFO  : OK
+--------------------------+--+
|           _c0            |
+--------------------------+--+
| ["2017-04-28 00:00:00"]  |
+--------------------------+--+
```